src/neuroconv/datainterfaces/ophys/miniscope/miniscopeconverter.py

```diff
@@ -0,0 +1,73 @@
+from typing import Literal
+
+from pynwb import NWBFile
+
+from ... import MiniscopeBehaviorInterface, MiniscopeImagingInterface
+from ....nwbconverter import NWBConverter
+from ....utils import FolderPathType, get_schema_from_method_signature
+
+
+class MiniscopeConverter(NWBConverter):
+    """Primary conversion class for handling Miniscope data streams."""
+
+    @classmethod
+    def get_source_schema(cls):
+        return get_schema_from_method_signature(cls)
+
+    @classmethod
+    def get_conversion_options_schema(cls):
+        return MiniscopeImagingInterface.get_conversion_options_schema(MiniscopeImagingInterface)
```

Comment:
```suggestion
    def get_conversion_options_schema(cls):
        return self.data_interface_objects["MiniscopeImaging"].get_conversion_options_schema()
```

Bah, just remembered this isn't a class method at all anymore; may have to fix my suggestion on the key